### ISSUE TITLE
SuperTokens list command breaks when base_path is set

### ISSUE THREAD

**Issue by JeremyEastham** [supertokens/supertokens-core#408]
## 🐛 Bug Report

When base_path is set in config.yaml, the supertokens list command fails with a 404 Not Found error:
```html
io.supertokens.cli.httpRequest.HTTPResponseException: Http error. Status Code: 404. Message: <!doctype html><html lang="en"><head><title>HTTP Status 404 â?? Not Found</title><style type="text/css">h1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} h2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} h3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} body {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} b {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} p {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;} a {color:black;} a.name {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 404 â?? Not Found</h1><hr class="line" /><p><b>Type</b> Status Report</p><p><b>Message</b> Not found</p><p><b>Description</b> The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.</p><hr class="line" /><h3>Apache Tomcat/8.5.47</h3></body></html>
        at io.supertokens.cli.httpRequest.HTTPRequest.sendGETRequest(HTTPRequest.java:100)
        at io.supertokens.cli.processes.Processes$RunningProcess.fetchConfigFilePath(Processes.java:48)
        at io.supertokens.cli.commandHandler.list.ListHandler.doCommand(ListHandler.java:35)
        at io.supertokens.cli.commandHandler.CommandHandler.handleCommand(CommandHandler.java:31)
        at io.supertokens.cli.Main.start(Main.java:101)
        at io.supertokens.cli.Main.main(Main.java:49)

Could not execute list command.
exiting
```

## Useful informations

Tested on Windows 11

How to reproduce:
1. Open an Administrator command prompt
2. Run `supertokens list` -> `No SuperTokens instances running`
3. Ensure `base_path` in config is commented out
4. Run `supertokens start`
5. Run `supertokens list` -> (proper output)
6. Run `supertokens stop`
7. Modify `base_path` in config to `/your/base/path/here`
8. Run `supertokens start`
9. Run `supertokens list` -> (404 error)

**Comment by JKEKBE** [comment]
Upon inspection of the code, this appears to be caused by `io.supertokens.cli.processes.RunningProcess::fetchConfigFilePath()`, which fires a GET request to `"http://" + this.hostName + ":" + this.port + "/config"`, which doesn't take into account the `base_path` in the config. Since the location of the config file is not known at this point in the code, perhaps it could be stored in the `.started` file. Currently, the `.started` file only stores the PID of the core on a single line, but it could store the config file path on a new line, or separated by some other separator.

**Comment by JeremyEastham** [comment]
Fixed by #412
